### PR TITLE
Add cosign verify functionality.

### PR DESCRIPTION
**Pull request by amartin120**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds cosign verify function when adding images to the hauler store.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: pkg/artifacts/file/getter/getter_test.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+		},
+		{
+			name: "should identify a directory",
+			args: args{
+				source: rootDir,
+			},
+			want: "directory",
+		},
+		{
+			name: "should identify a http",
```
Comment (zackbradys) [inline pkg/artifacts/file/getter/getter_test.go]:
```suggestion
			name: "should identify an http fqdn",
```

File: pkg/artifacts/file/getter/getter_test.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+			args: args{
+				source: "http://my.cool.website",
+			},
+			want: "http",
+		},
+		{
+			name: "should identify a http",
```
Comment (zackbradys) [inline pkg/artifacts/file/getter/getter_test.go]:
```suggestion
			name: "should identify an http fqdn",
```

File: pkg/consts/consts.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+	// FileLayerMediaType is the reserved media type for File content layers
+	FileLayerMediaType = "application/vnd.content.hauler.file.layer.v1"
+
+	// FileLocalConfigMediaType is the reserved media type for File config
+	FileLocalConfigMediaType     = "application/vnd.content.hauler.file.local.config.v1+json"
+	FileDirectoryConfigMediaType = "application/vnd.content.hauler.file.directory.config.v1+json"
+	FileHttpConfigMediaType      = "application/vnd.content.hauler.file.http.config.v1+json"
+
+	// MemoryConfigMediaType
```
Comment (zackbradys) [inline pkg/consts/consts.go]: add description to "MemoryConfigMediaType is ..." to match others?

**Comment by mgogoi** [comment]
Seems to have merge conflicts @amartin120

**Review by dweomer — APPROVED** [review]

**Pull request merged**